lib/sord/logging.rb

```diff
@@ -74,7 +74,8 @@ def self.generic(kind, header, msg, item, **opts)
       return unless enabled_types.include?(kind)
```

Comment:
The "important warnings" summary does not include these new paths - is it straightforward to add them here too?

If it's a faff, don't worry about it - not a big deal.

![CleanShot 2025-03-03 at 20 00 07@2x](https://github.com/user-attachments/assets/f8fbf15c-c377-40cf-8911-0811fdaeb53e)

Reply:
Sure, can take a look in maybe an hour here.  Could you help me understand the goal in repeating these log entries?

Reply:
I can't speak for everyone's IDE, but my editor needs to be specifically told how to parse file/line numbers out of non-standard formats (that was why I had chosen the format of filename:line: at the beginning of the file - it's generally parsed by default).  If I could understand the rationale for the summary, it might help me come up with an idea on how to keep people from having to do that twice, as this format is different than the one we use in normal logging.

Reply:
Thanks!

Sord's decision to use WARN means "there's a solid chance this could cause a type error" - as opposed to OMIT, where we can fall back to `T.untyped`, or INFER, where we can have a reasonable guess at what you meant.

So, if there are any warnings, Sord repeats them so they don't get lost in the rest of the output.

(The constant cases in that screenshot are less likely to be a problem, if your YARD docs aren't aware of some library. Others - like type parsing errors - can deserve some attention.)

Reply:
Change is pushed up - take a look!

Reply:
Amazing stuff, thanks!